plugins/VaccineAppointmentScanner/__init__.py

```diff
@@ -0,0 +1,119 @@
+from nextcord.ext import commands, tasks
+from Plugin import AutomataPlugin
+
+import requests
+
+## Mapping of major St. John's vaccine location IDs to location names
+LOCATIONS = {
+    949: "The Village Mall YELLOW",
+    943: "The Village Mall BLUE",
+    776: "FORMER Public Service Commission Building (50 Mundy Pond Road) (Ages <29)",
+    1668: "The Village Mall RED",
+    1669: "The Village Mall GREEN",
+    814: "Reid Community Centre (Location A) (Ages 30+)",
+    1672: "Reid Community Centre (Location B) (Ages 30+)",
+    1673: "Reid Community Centre (Location C) (Ages 30+)"
+}
+
+## The starting page for the vaccine booking system. Used to get a valid session cookie.
+START_URL = "https://portal.healthmyself.net/nleasternhealth/forms/P2E"
+
+## The first page for the vaccine booking process. Used to get a valid session cookie.
+BOOKING_URL = "https://portal.healthmyself.net/nleasternhealth/guest/booking/form/abec68ea-5a99-421b-b137-6e83cf7a3231"
+
+## The endpoint for accessing available appointment data.
+LOCATION_URL = "https://portal.healthmyself.net/nleasternhealth/guest/booking/type/5892/locations"
+
+class Appointments(AutomataPlugin):
+
```

Comment:
A little doc string here """ The plugin does """ for when the !help Appointments is run it'll show a nice little embed like so 
<img width="440" alt="image" src="https://user-images.githubusercontent.com/1349268/149037375-bccbb7ff-87e1-4fee-8a1f-b8516d6e79d8.png">

Without it nothing will happen, in the background the help command wrapper is expecting it.